.github/workflows/test_dev.yml

```diff
@@ -0,0 +1,77 @@
+name: Test dev
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
```

Comment:
Since these are equal to the triggers on test stable, they are going to always run in parallel. if you want test dev to only happen after stable, I think you need to remove this bit

Reply:
I'll test it. Thanks!